lib/compact.js

```diff
@@ -399,14 +415,14 @@ api.compact = ({
                 relativeTo: {vocab: true}
               })] = expandedItem['@index'];
             }
-          } else if(nestResult.hasOwnProperty(itemActiveProperty)) {
-            // can't use @list container for more than 1 list
-            throw new JsonLdError(
-              'JSON-LD compact error; property has a "@list" @container ' +
-              'rule but there is more than a single @list that matches ' +
-              'the compacted term in the document. Compaction might mix ' +
-              'unwanted items into the list.',
-              'jsonld.SyntaxError', {code: 'compaction to list of lists'});
+          // FIXME
+          //} else if(nestResult.hasOwnProperty(itemActiveProperty)) {
```

Comment:
@gkellogg Can this FIXME and conditional be removed?  I think I was adapting the ruby code changes and wasn't sure what to do here.

Reply:
My code doesn't have such a case, and it doesn't seem to be necessary for passing tests; I don't know what the original intention was for.